src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -77,7 +77,9 @@ def list_express_route_circuits(cmd, resource_group_name=None):
 
 def create_express_route_auth(cmd, resource_group_name, circuit_name, authorization_name):
     client = network_client_factory(cmd.cli_ctx).express_route_circuit_authorizations
-    return client.create_or_update(resource_group_name, circuit_name, authorization_name)
+    ExpressRouteCircuitAuthorization = cmd.get_models('ExpressRouteCircuitAuthorization')
```

Comment:
`ExpressRouteCircuitAuthorization` has two fields. I tried to expose these two in our command UX, however, during my test I cannot set any of them. The server will set both of two properties.